src/hearings/containers/linked-hearings/linked-hearings-how-to-heard/linked-hearings-how-to-heard.component.ts

```diff
@@ -0,0 +1,177 @@
+import { AfterViewInit, Component, OnDestroy, OnInit } from '@angular/core';
+import { FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { HearingLinksStateData } from '../../../models/hearingLinksStateData.model';
+import { ACTION, GroupLinkType } from '../../../models/hearings.enum';
+import { LinkedHearingGroupMainModel, ServiceLinkedCasesModel } from '../../../models/linkHearings.model';
+import { HearingsService } from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+import { ValidatorsUtils } from '../../../utils/validators.utils';
+import { RequestHearingPageFlow } from '../../request-hearing/request-hearing.page.flow';
+
+@Component({
+  selector: 'exui-linked-hearings-how-to-heard',
+  templateUrl: './linked-hearings-how-to-heard.component.html',
+  styleUrls: ['./linked-hearings-how-to-heard.component.scss']
+})
+export class HowLinkedHearingsBeHeardComponent
+  extends RequestHearingPageFlow
```

Comment:
do we need this RequestHearingPageFlow? it should be using the hearing actual way to navigate pages.